pkg/bootstrapper/bootstrapper.go

```diff
@@ -522,6 +526,9 @@ func (wmcb *winNodeBootstrapper) generateInitialKubeletArgs(args map[string]stri
 	if nodeWorkerLabel, ok := args["node-labels"]; ok {
 		kubeletArgs = append(kubeletArgs, "--"+"node-labels"+"="+nodeWorkerLabel)
 	}
+	if wmcb.nodeIP != "" {
+		kubeletArgs = append(kubeletArgs, "--node-ip="+wmcb.nodeIP)
```

Comment:
Currently it's not clear to me where `nodeIP` is being set. I assume the value will be passed in by WMCO when it runs `wmcb.exe --node-ip= ...`?

Reply:
Yes that will passed in via WMCO when running WMCB